src/orion/client/experiment.py

```diff
@@ -383,20 +447,55 @@ def suggest(self):
             Reserved trial for execution. Will return None if experiment is done or broken
             of if the algorithm cannot suggest until other trials complete.
 
+        Raises
+        ------
+        `WaitingForTrials`
+            if the HPO has not finished but no trials are available at the moment
```

Comment:
```suggestion
            if the experiment is not completed and algorithm needs to wait for some trials to complete 
            before it can suggest new trials.
```